swagger/v3/api_spec.json

```diff
@@ -161,12 +161,12 @@
         }
       }
     },
-    "/api/v3/partnerships/ecf": {
```

Comment:
could we avoid running swaggerise on v3, and amending manually, as this is the only version that has not been fixed properly, and will change up the whole file?

Reply:
can we please avoid this large change, can we jump on a call?